plugins/huaweicloud/src/main/java/org/eclipse/xpanse/plugins/huaweicloud/manager/HuaweiCloudManagerService.java

```diff
@@ -0,0 +1,104 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ * SPDX-FileCopyrightText: Huawei Inc.
+ *
+ */
+
+package org.eclipse.xpanse.plugins.huaweicloud.manager;
+
+import com.huaweicloud.sdk.core.auth.ICredential;
+import com.huaweicloud.sdk.ecs.v2.EcsClient;
+import com.huaweicloud.sdk.ecs.v2.model.BatchRebootServersRequest;
+import com.huaweicloud.sdk.ecs.v2.model.BatchRebootServersResponse;
+import com.huaweicloud.sdk.ecs.v2.model.BatchStartServersRequest;
+import com.huaweicloud.sdk.ecs.v2.model.BatchStartServersResponse;
+import com.huaweicloud.sdk.ecs.v2.model.BatchStopServersRequest;
+import com.huaweicloud.sdk.ecs.v2.model.BatchStopServersResponse;
+import lombok.extern.slf4j.Slf4j;
+import org.eclipse.xpanse.modules.credential.CredentialCenter;
+import org.eclipse.xpanse.modules.models.credential.AbstractCredentialInfo;
+import org.eclipse.xpanse.modules.models.credential.enums.CredentialType;
+import org.eclipse.xpanse.modules.models.service.common.enums.Csp;
+import org.eclipse.xpanse.modules.orchestrator.manager.ServiceManagerRequest;
+import org.springframework.http.HttpStatus;
+import org.springframework.stereotype.Component;
+
+/**
+ * Class that encapsulates all Manager-related public methods of the Huawei Cloud plugin.
+ */
+@Slf4j
+@Component
+public class HuaweiCloudManagerService {
+
+    private final CredentialCenter credentialCenter;
+    private final HuaweiCloudManagerClient huaweiCloudManagerClient;
+    private final HuaweiCloudManagerConverter huaweiCloudManagerConverter;
+
+    /**
+     * Constructs a HuaweiCloudManagerService with the necessary dependencies.
+     */
+    public HuaweiCloudManagerService(
+            CredentialCenter credentialCenter,
+            HuaweiCloudManagerClient huaweiCloudManagerClient,
+            HuaweiCloudManagerConverter huaweiCloudManagerConverter) {
+        this.credentialCenter = credentialCenter;
+        this.huaweiCloudManagerClient = huaweiCloudManagerClient;
+        this.huaweiCloudManagerConverter = huaweiCloudManagerConverter;
+    }
+
+    /**
+     * Start the Huawei Cloud Ecs server.
+     */
+    public boolean startService(ServiceManagerRequest serviceManagerRequest) {
+        EcsClient ecsClient = getEcsClient(serviceManagerRequest);
+        BatchStartServersRequest request =
+                huaweiCloudManagerConverter.buildBatchStartServersRequest(
+                        serviceManagerRequest.getDeployResourceEntityList());
+        BatchStartServersResponse batchStartServersResponse = ecsClient.batchStartServers(request);
```

Comment:
these method are not synchornous. It will return a OK immediatly but does not mean the action is successful. We must poll the response. 
Check the 2nd example here - https://github.com/huaweicloud/huaweicloud-sdk-java-v3#723-typical-usage-scenarios-top
We must do something like this.